src/account/interface.cairo

```diff
@@ -25,31 +25,33 @@ trait IDeclarer<TState> {
     fn __validate_declare__(self: @TState, class_hash: felt252) -> felt252;
 }
 
+//
+// Account ABI
+//
+
 #[starknet::interface]
 trait AccountABI<TState> {
+    // ISRC6
     fn __execute__(self: @TState, calls: Array<Call>) -> Array<Span<felt252>>;
     fn __validate__(self: @TState, calls: Array<Call>) -> felt252;
+    fn is_valid_signature(self: @TState, hash: felt252, signature: Array<felt252>) -> felt252;
+
+    // ISRC5
+    fn supports_interface(self: @TState, interface_id: felt252) -> bool;
+
+    // IDeclarer
     fn __validate_declare__(self: @TState, class_hash: felt252) -> felt252;
+
+    // Non-Standard
     fn __validate_deploy__(
         self: @TState, class_hash: felt252, contract_address_salt: felt252, _public_key: felt252
     ) -> felt252;
     fn set_public_key(ref self: TState, new_public_key: felt252);
     fn get_public_key(self: @TState) -> felt252;
-    fn is_valid_signature(self: @TState, hash: felt252, signature: Array<felt252>) -> felt252;
-    fn supports_interface(self: @TState, interface_id: felt252) -> bool;
-}
 
-// Entry points case-convention is enforced by the protocol
-#[starknet::interface]
-trait AccountCamelABI<TState> {
-    fn __execute__(self: @TState, calls: Array<Call>) -> Array<Span<felt252>>;
-    fn __validate__(self: @TState, calls: Array<Call>) -> felt252;
-    fn __validate_declare__(self: @TState, classHash: felt252) -> felt252;
-    fn __validate_deploy__(
-        self: @TState, classHash: felt252, contractAddressSalt: felt252, _publicKey: felt252
-    ) -> felt252;
-    fn setPublicKey(ref self: TState, newPublicKey: felt252);
-    fn getPublicKey(self: @TState) -> felt252;
+    // Account Camel
```

Comment:
```suggestion
    // Camel case compatibility
```
not sure if it's worth adding a link to the interface migration docs

Reply:
It's probably not a bad idea to include a link. I'll add it and see what we think